xclim/data/fr.json

```diff
@@ -674,7 +702,14 @@
     "description": "Jour de l'année de la fin de la saison de végétation. La saison de végétation est définie comme l'intervalle entre la première série de {window} jours où la température journalière est au-dessus de {thresh} et la première série (après le {mid_date}) de {window} jours où elle est sous {thresh}.",
     "comment": "",
     "title": "Fin de la saison de végétation",
-    "abstract": "Jour de l'année de la fin de la saison de végétation. La saison de végétation est définie comme l'intervalle entre la première série de N jours où la température journalière est au-dessus d'un seuil et la première série (après une certaine date) de N jours où elle est sous ce même seuil"
+    "abstract": "Jour de l'année de la fin de la saison de végétation. La saison de végétation est définie comme l'intervalle entre la première série de N jours où la température journalière est au-dessus d'un seuil et la première série (après une certaine date) de N jours où elle est sous ce même seuil."
+  },
+  "FROST_FREE_SEASON_END": {
+    "long_name": "Jour de l'année de la fin de la saison sans gel",
+    "description": "Jour de l'année de la fin de la saison sans gel. La saison sans gél est définie comme l'intervalle entre la première série de {window} jours où la température journalière est au-dessus de {thresh} et la première série (après le {mid_date}) de {window} jours où elle est sous {thresh}.",
+    "comment": "",
+    "title": "Fin de la saison sans gél",
+    "abstract": "Jour de l'année de la fin de la saison sans gél. La saison sans gél est définie comme l'intervalle entre la première série de N jours où la température journalière est au-dessus d'un seuil et la première série (après une certaine date) de N jours où elle est sous ce même seuil."
```

Comment:
```suggestion
    "description": "Jour de l'année de la fin de la saison sans gel. La saison sans gel est définie comme l'intervalle entre la première série de {window} jours où la température journalière est au-dessus de {thresh} et la première série (après le {mid_date}) de {window} jours où elle est sous {thresh}.",
    "comment": "",
    "title": "Fin de la saison sans gel",
    "abstract": "Jour de l'année de la fin de la saison sans gél. La saison sans gel est définie comme l'intervalle entre la première série de N jours où la température journalière est au-dessus d'un seuil et la première série (après une certaine date) de N jours où elle est sous ce même seuil."
```

Reply:
The texts for `frost_free_season_start` and  `frost_free_season_end` are quite different, is there a reason for this?

Reply:
I think I based them on existing indices, I can try and synchronize them for good measure.